app/homepage/index.js

```diff
@@ -73,13 +73,53 @@ const getUsername = () => {
 		.then(({ username }) => {
 			document.getElementById(
 				'logged-in'
-			).textContent = `${username}, welcome to NASA`;
+			).textContent = `Welcome to your favourite astronomy page, ${username}!`;
+			document.getElementById('login-link').remove();
+			document.getElementById('register-link').remove();
+
+			const roverMapList = document.getElementById('navbar');
+
+			const roverMapLink = document.createElement('a');
+			roverMapLink.href = '/user/rover';
+			roverMapLink.textContent = 'Rover';
+			roverMapList.appendChild(roverMapLink);
+
+			const calculatorLink = document.createElement('a');
+			calculatorLink.href = '/user/calculator';
+			calculatorLink.textContent = 'Calculator';
+			roverMapList.appendChild(calculatorLink);
+
+			const userAccountLink = document.createElement('a');
+			userAccountLink.href = '/user/account';
+			userAccountLink.textContent = 'Account';
+			roverMapList.appendChild(userAccountLink);
+
+			const logoutLink = document.createElement('a');
+			logoutLink.href = '/user/logout';
+			logoutLink.textContent = 'Logout';
+			roverMapList.appendChild(logoutLink);
 		})
 		.catch((error) => {
 			console.log(error);
 		});
 };
 
+const stickyNavBar = () => {
+	const navbar = document.getElementById('navbar');
+	const offsetPositionOfNavBar = navbar.offsetTop;
+
+	const toggleStickyClass = () => {
+		if (window.pageYOffset >= offsetPositionOfNavBar) {
+			navbar.classList.add('sticky');
+		} else {
+			navbar.classList.remove('sticky');
+		}
+	};
```

Comment:
My suspicion of what would happen is that when you start scrolling, the rest of the page will jump up a number of pixels equal to the height of the navbar. Is that true? Have you already resolved this problem? Would I be able to take a look?

Reply:
Fixed. Thank you.